ext/pgsql/tests/gh17165.phpt

```diff
@@ -0,0 +1,65 @@
+--TEST--
+Fix pg_query()/pg_query_params()/pg_prepare()/pg_execute()/pg_set_error_verbosity()/pg_set_client_encoding()/pg_put_line() pg field infos calls ArgumentCountError message.
+--EXTENSIONS--
+pgsql
+--SKIPIF--
+<?php
+include("skipif.inc");
```

Comment:
This should better be an absolute path.

Reply:
The reason why the test fails on PHP-8.4 and master is that skipif.inc has been moved to inc/skipip.inc (same with config.inc), though.

Reply:
yes I forgot this, just tested the build as I needed change for higher branches.